glusterd2/commands/volumes/volume-start.go

```diff
@@ -69,6 +73,12 @@ func volumeStartHandler(w http.ResponseWriter, r *http.Request) {
 	logger := gdctx.GetReqLogger(ctx)
 	volname := mux.Vars(r)["volname"]
 
+	query := r.URL.Query().Get("force")
+	force, err := strconv.ParseBool(query)
+	if err != nil {
+		restutils.SendHTTPError(ctx, w, http.StatusBadRequest, errors.ErrInvalidQuery)
```

Comment:
`ErrInvalidQuery` is very generic and not useful. Do not define a constant for that.

Instead, pass error string here in the handler.

```go
restutils.SendHTTPError(ctx, w, http.StatusBadRequest, "The query param force only accepts boolean values")
```

Reply:
done